### PR TITLE
Portfolio: Windows PopUp

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
## Portfolio: Windows PopUp

In this project I:

- implement a popup window that includes the project details in both desktop and mobile
- use a JavaScript array to store all of the information for all projects.

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+for (let index = 0; index < hamItem.length; index += 1) {
+  hamItem[index].addEventListener('click', () => {
+    closeNav();
+  });
+}
+
+const workDiv = document.querySelector('.work');
+
+const projects = [
```
Comment (carlos-ssh) [inline assets/js/script.js:28]: - Kindly provide the requested style by the design guidelines
Screen Shot 2021-10-14 at 15 03 22

File: index.html [hunk 2]
```diff
@@ -1,54 +1,162 @@
...
-          <a href="#"><img src="assets/images/mobile/Icon-Menu.svg" alt="Menue"></a>
-        </div>
-      </div>
+    <div class="logo">
+      <a href="#"><img src="assets/images/mobile/My-Logo.svg" alt="Portfolio-Logo"></a>
+    </div>
+    <div class="menu-icon" onclick="openNav()" id="click-image">
+      <a href="#"><img src="assets/images/mobile/Icon-Menu.svg" alt="Navbar-Menue"></a>
+    </div>
+    <nav id="myNav" class="overlay">
```
Comment (carlos-ssh) [inline index.html:32]: - Kindly work on your design... Your mobile menu are appearing outside it:
Screen Shot 2021-10-14 at 15 05 12

File: assets/js/script.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+IconMenu.addEventListener('click', () => {
+  openNav();
+});
+
+for (let index = 0; index < hamItem.length; index += 1) {
+  hamItem[index].addEventListener('click', () => {
+    closeNav();
+  });
+}
+
```
Comment (carlos-ssh) [inline assets/js/script.js:25]: All your images are broken kindly work on it:
Screen Shot 2021-10-14 at 15 07 18

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
@Emmy-github-webdev, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._




changesrequested

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ If you like what you see and have a project you need coded, don't hesitate to co
 
 # Live Demo
 
-[Live Demo Link](https://github.com/Emmy-github-webdev/my-portfolio).
+[Live Demo Link](https://emmy-github-webdev.github.io/my-portfolio/).
 
 # Getting Started
```
Comment (carlos-ssh) [inline README.md:20]: - PROVIDE A LICENSE.

File: assets/js/script.js [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+    const tagDiv = document.createElement('div');
+    tagDiv.className = 'tag';
+    for (let j = 0; j < project.technologies.length; j += 1) {
+      const tagDivSpan = document.createElement('span');
+      tagDivSpan.textContent = project.technologies[j];
+      tagDiv.appendChild(tagDivSpan);
+    }
+    const createProjectBtn = document.createElement('a');
+    createProjectBtn.className = 'see-proj-btn';
+    createProjectBtn.textContent = 'See Project';
```
Comment (SajjadAhmad14) [inline assets/js/script.js:123]: Pop-up on the desktop is still missing key design elements. Please use the same font as mentioned in the design. The buttons on the right side are overlapping. Moreover please make the main content on background blur as can be seen in the design.
pop-desktop

Comment (SajjadAhmad14) [inline assets/js/script.js:123]: Buttons are overflowing out of the pop-up on mobile view. Please follow the font guidelines from the design.
pop-mobile

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Emmanuel, 
 ### Thank you for working on changes suggested previously :ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please adjust the picture to match the original it should be on the same line as the text. Check picture:
Screenshot from 2021-10-15 20-24-52

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**